cfme/infrastructure/config_management/__init__.py

```diff
@@ -414,32 +389,35 @@ def rest_api_entity(self):
             name=self.ui_name
         ).provider_id
 
-        return RestEntity(
-            self.appliance.rest_api.collections.providers,
-            data={
-                "href": self.appliance.url_path(
-                    "/api/providers/{}?provider_class=provider".format(provider_id)
-                )
-            },
-        )
+        return RestEntity(self.appliance.rest_api.collections.providers, data={
+            "href": self.appliance.url_path(
+                "/api/providers/{}?provider_class=provider".format(provider_id)
+            )
+        })
 
+    # TODO: implement this via Sentaku
     def create_rest(self):
         """Create the config manager in CFME using REST"""
+        include_ssl = False
         if "ansible_tower" in self.key:
```

Comment:
As we have `type` as attribute; Is it possible to use it (self.type) for this check?